src/App.js

```diff
@@ -36,23 +45,16 @@ const App = () => {
           style={{ overflow: 'scroll', maxHeight: '1000px', marginTop: '100px' }}
         >
           <Grid xs={12}>
-            <ViewThemeDialog
-              theme={state.theme}
-            />
+            {createElement(AccessToken, { ...sharedProps, setState })}
```

Comment:
do we need to do this or can we just do < AccessToken {...sharedProps, setState} />